src/tests/ftest/ior/ior_intercept_messages.yaml

```diff
@@ -21,25 +21,22 @@ container:
     type: POSIX
     control_method: daos
 ior:
+    api: POSIX
     client_processes:
         np_16:
             np: 16
     test_file: daos:testFile
     repetitions: 1
     iorflags:
         ssf:
-          flags: "-v -D 10 -w -r"
-          ior_api:
-            - POSIX
-          transfersize_blocksize:
-            1M:
-              transfer_size: '1M'
-              block_size: '100M'
-              write_x: 2
-              read_x: 1
-          objectclass:
-            oclass_SX:
-              dfs_oclass: "SX"
+            flags: "-v -w -r"
```

Comment:
Minor: `flags` can be brought out of 'ssf' as well and 'ssf' can be removed altogether.

Reply:
Fixed